coordinator/src/services/translation/prescription/practitioner.ts

```diff
@@ -40,7 +41,9 @@ export function convertResponsibleParty(
   fhirBundle: fhir.Bundle,
   fhirMedicationRequest: fhir.MedicationRequest,
   isCancellation: boolean,
-  convertPractitionerRoleFn = convertPractitionerRole
+  convertPractitionerRoleFn = convertPractitionerRole,
```

Comment:
Besides the id function, none of these defaults ever seem to be overridden, even in tests. Might be nice to get rid of these parameters

Reply:
Agreed